troposphere/static/js/AppRoutes.react.js

```diff
@@ -66,22 +67,22 @@ define(function (require) {
       </Route>
 
       <Route name="providers" handler={ProvidersMaster}>
-        <Route name="provider" path=":providerId" handler={ProviderDetailsPage}/>
-        <DefaultRoute handler={ProviderDetailsPage}/>
+        <DefaultRoute handler={ProviderListSection} />
+        <Route name="provider" path=":id" handler={ProviderDetail}/>
+        <Route name="all-providers" path="/" handler={ProviderListSection} />
       </Route>
 
       <Route name="help" handler={HelpPage}/>
       <Route name="settings" handler={SettingsPage}/>
 
       <Route name="admin" handler={AdminMaster}>
-        <Route name="identity-membership-manager" path="users" handler={IdentityMembershipMaster}/>
+        <DefaultRoute name="identity-membership-manager" path="users" handler={IdentityMembershipMaster}/>
```

Comment:
Okay - Admin has lower priority for consideration since the number of people seeing it will be significantly lower.  But we can definitely look at this.